pkg/envoy/get_process_attr_windows.go

```diff
@@ -0,0 +1,12 @@
+//go:build windows
+// +build windows
+
+package envoy
+
+import "syscall"
+
+func getProcessAttr() *syscall.SysProcAttr {
+	return &syscall.SysProcAttr{
+		CreationFlags: syscall.CREATE_NEW_PROCESS_GROUP,
+	}
+}
```

Comment:
@pglass  i had to create a seperate windows and linux versions of this syscall to create process group. should i mention this in changelog ?

Reply:
I don't think it's necessary to call this out in the changelog.